packages/es-indexer/src/indexers/TrackIndexer.ts

```diff
@@ -49,10 +49,13 @@ export class TrackIndexer extends BaseIndexer<TrackDoc> {
         has_stems: { type: 'boolean' },
         bpm: { type: 'float' },
         musical_key: { type: 'keyword' },
+        play_count: { type: 'integer' },
 
         // saves
         saved_by: { type: 'keyword' },
         save_count: { type: 'integer' },
+        favorite_count: { type: 'integer' },
```

Comment:
@stereosteve this is the part that feels weird

Reply:
isn't `save_count` the same as `favorite_count`?

`save_count` is the name of the field in the sql database... so we should prefer that.

Reply:
okay good to know, lower down we compute `favorite_count` which had me a bit confused, and if you look at the api response, we actually send down `favorite_count`: https://discoveryprovider2.staging.audius.co/v1/full/tracks?id=7eP5n. (note on the front end we convert it back to save_count)